db/migrate/20150204174650_add_admin_editable_to_price_group.rb

```diff
@@ -5,9 +5,11 @@ class AddAdminEditableToPriceGroup < ActiveRecord::Migration[4.2]
   def up
     add_column :price_groups, :admin_editable, :boolean, after: :is_internal, null: false, default: true
 
-    (PriceGroup.globals - [PriceGroup.cancer_center]).each do |price_group|
-      price_group.update_attribute(:admin_editable, false)
-    end
+    # This was relevant for updating records when the migration was run back in 2015.
+    # The PriceGroup#cancer_center method no longer exists.  Leaving this here for documentation.
```

Comment:
@jhanggi what do you think about this?  I could see an argument for leaving it alone or removing it altogether.  Let me know what you think.

Reply:
I think this is probably a good call to leave it in. The seeds cover the right settings for this attribute, right?